src/planwise/client/projects/handlers.cljs

```diff
@@ -194,26 +194,81 @@
                     (assoc-in [:facilities :isochrones] nil))
          filters (db/project-filters new-db)
          project-id (get-in db [:project-data :id])]
-     (api/update-project project-id filters :facilities
-                         :projects/project-updated)
-     new-db)))
+     (transition-map-state new-db :facilities))))
+
+(defn transition-map-state [db request-with]
```

Comment:
This name is not clear as to what's the intention of the function. Maybe `start-update-request` or something along those lines?